### ISSUE TITLE
GH Artifact provider: Unable to retrieve artifact list for revision

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/craft#182]
I'm trying to release [Symbol Collector 1.2.2](https://github.com/getsentry/symbol-collector/actions/runs/585018062) and it's failing while processing the artifact.

[No change to how the artifact is build change since the previous version 1.2.1](https://github.com/getsentry/symbol-collector/compare/1.2.1...main).

It seems zlib related (log below) but I downloaded the artifact locally and could extract files without any problems.

The issue affects version `0.16.1`. I upgraded to the latest `0.17.2` and received the same error.

I've received two different errors that alternate (somewhat randomly) with each other if I retry the command:

```
ℹ info [artifact-provider/github] › Fetching Github artifacts for getsentry/symbol-collector, revision 0dd644d0d67336217dada5da81c9e1b5fa0aabed
ℹ info [artifact-provider/github] › Requesting archive URL from Github...
ℹ info [artifact-provider/github] › Downloading ZIP from Github artifacts...
ℹ info [artifact-provider/github] › Finished downloading.
ℹ info [artifact-provider/github] › Extracting "/var/folders/3r/9m5hqv695t97rd0qllgcbyjc0000gn/T/craft-601315Z0kfxUZaeb3" to "/var/folders/3r/9m5hqv695t97rd0qllgcbyjc0000gn/T/craft-7b1FS2"...
✖ error [artifact-provider] › Unable to retrieve artifact list for revision 0dd644d0d67336217dada5da81c9e1b5fa0aabed!
✖ error Error: unexpected end of file
  at Zlib.zlibOnError [as onerror] (zlib.js:182:17)
```

And
```
ℹ info [artifact-provider/github] › Fetching Github artifacts for getsentry/symbol-collector, revision 0dd644d0d67336217dada5da81c9e1b5fa0aabed
ℹ info [artifact-provider/github] › Requesting archive URL from Github...
ℹ info [artifact-provider/github] › Downloading ZIP from Github artifacts...
ℹ info [artifact-provider/github] › Finished downloading.
ℹ info [artifact-provider/github] › Extracting "/var/folders/3r/9m5hqv695t97rd0qllgcbyjc0000gn/T/craft-60601P38ruEZ8pQlz" to "/var/folders/3r/9m5hqv695t97rd0qllgcbyjc0000gn/T/craft-9Xn1HL"...
✖ error [artifact-provider] › Unable to retrieve artifact list for revision 0dd644d0d67336217dada5da81c9e1b5fa0aabed!
✖ error Error: invalid signature: 0x6f63227b
  at /Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/unzipper/lib/parse.js:59:26
  at tryCatcher (/Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/bluebird/js/release/util.js:16:23)
  at Promise._settlePromiseFromHandler (/Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/bluebird/js/release/promise.js:510:31)
  at Promise._settlePromise (/Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/bluebird/js/release/promise.js:567:18)
  at Promise._settlePromise0 (/Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/bluebird/js/release/promise.js:612:10)
  at Promise._settlePromises (/Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/bluebird/js/release/promise.js:691:18)
  at Async._drainQueue (/Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/bluebird/js/release/async.js:133:16)
  at Async._drainQueues (/Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/bluebird/js/release/async.js:143:10)
  at Immediate.Async.drainQueues (/Users/bruno/.volta/tools/image/packages/@sentry/craft/0.16.1/node_modules/bluebird/js/release/async.js:17:14)
  at processImmediate (internal/timers.js:461:21)
```

**Comment by BYK** [comment]
@bruno-garcia this should be resolved with #177 but it is not released yet. Are you able to try from latest master?

Also, what would it take to move Symbol Collector to `getsentry/publish` so you are not releasing from your local machine?

**Comment by bruno-garcia** [comment]
> @bruno-garcia this should be resolved with #177 but it is not released yet. Are you able to try from latest master?

I can wait for a new version of craft to go out.
 
> Also, what would it take to move Symbol Collector to `getsentry/publish` so you are not releasing from your local machine?

I don't know, I haven't put any project on `getsentry/publish` yet.

**Comment by BYK** [comment]
@bruno-garcia this fix is now live with the `0.18.0` version. Can you verify that it is fixed (and close the issue if it is)?

>I don't know, I haven't put any project on getsentry/publish yet.

Happy to collaborate there.

**Comment by bruno-garcia** [comment]
> @bruno-garcia this fix is now live with the `0.18.0` version. Can you verify that it is fixed (and close the issue if it is)?

Thanks! I have no plans to release symbol collector so I'll close this for now and reopen if the problem persists.
> 
> > I don't know, I haven't put any project on getsentry/publish yet.
> 
> Happy to collaborate there.

Great, I'll reach out soon as we start moving repos. For JS related repos it should just work but for other repos there'll be more requirements in terms of tools available on the build box (i.e `dotnet`, `flutter`)

**Comment by BYK** [comment]
I'm going to reopen as I found https://github.com/getsentry/publish/issues/140#issuecomment-785829177 which happened with Craft@0.18.0

Need to investigate more.
